### ISSUE TITLE
Warning: DOMNode::cloneNode(): ID <id> already defined in phar:

### ISSUE THREAD

**Issue by localgod** [pdepend/pdepend#197]
I get a couple of these warnings:
```
Warning: DOMNode::cloneNode(): ID jdepend.good already defined in phar:///Users/.../pdepend.phar/src/main/php/PDepend/Report/Jdepend/Chart.php on line 199
Warning: DOMNode::cloneNode(): ID jdepend.legend already defined in phar:///Users/.../pdepend.phar/src/main/php/PDepend/Report/Jdepend/Chart.php on line 224
Warning: DOMNode::cloneNode(): ID jdepend.bad already defined in phar:///Users/.../pdepend.phar/src/main/php/PDepend/Report/Jdepend/Chart.php on line 201
```
Running the phar version (2.0.6)

**Comment by GHugo** [comment]
Confirmed.

**Comment by vikingjs** [comment]
If it helps, I'm getting the same warnings using php 5.6. Looks like several cases where cloning a node with an ID is raising the warning.

**Comment by uirapuru** [comment]
Same here.

**Comment by manuelpichler** [comment]
I cannot reproduce that error with php 5.6.8 under ubuntu linux. I would expect it has something todo with the underlying libxml2 version. Will try to fix it blind

**Comment by manuelpichler** [comment]
Could you try if the following phar release solves your problem?

**Comment by vikingjs** [comment]
Happy to test, but I used Composer to install pdepend. The way of the future, they say. Going to have to find a quiet moment to get my build working with the phar.

If it helps, the libxml2 that's installed on my machine is 2.9.2

**Comment by sataris** [comment]
I just managed to test the snapshot above and can confirm it runs with no errors

**Comment by sebastianbergmann** [comment]
I was experiencing the same issue on Fedora 22 and can confirm that PDepend 2.1.0snapshot201505210758 solves the issue.

**Comment by localgod** [comment]
PDepend 2.1.0snapshot201505210758 solves this issue on osx with php version 5.6.11

**Comment by hackel** [comment]
I tried the 201505210758 snapshot and it also solved the issue for me.  Unfortunately it looks like this change didn't make it into 2.1.  Any chance we can get a new release soon?  It's easier that trying to figure out how to add this one phar to my CI setup.

**Comment by mintbridge** [comment]
I have the same problem on 5.6.4 but not 5.6.10 or 5.6.11 without the snapshot

**Comment by topdk-jnz** [comment]
The update fixes issue running on Windows 7, php 5.6.11.

**Comment by manuelpichler** [comment]
Fixed in 2221f74
